graphql/src/main/java/io/micronaut/configuration/graphql/ws/apollo/GraphQLApolloWsConfiguration.java

```diff
@@ -88,6 +90,15 @@ public boolean isEnabled() {
         return enabled;
     }
 
+    /**
+     * Sets whether GraphQL websocket is enabled.
```

Comment:
make sure the javadoc which goes into the configuration reference documentation outputs the default value